index.html

```diff
@@ -9,11 +9,87 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
+<header>
```

Comment:
This `header` element should go in the `body`, not `head`. All HTML code goes in `body`. The `head` is for other things not HTML